.travis.yml

```diff
@@ -51,68 +51,24 @@ install:
         export PATH="/usr/local/opt/ccache/libexec:$PATH"
         && brew link --overwrite gcc;
     fi
-  - export CURR_DIR=$PWD
-  - cd $HOME/install
 # OCCA
-  - if [[ -d occa ]]; then
-        git -C occa pull --allow-unrelated-histories;
-        if [[ $? -eq 0 ]]; then
-          git -C occa reset --hard $OCCA_HEAD;
-        else
-          rm -rf occa
-          && git clone https://github.com/libocca/occa.git
-          && git -C occa reset --hard $OCCA_HEAD;
-        fi
-    else
-        git clone https://github.com/libocca/occa.git
+  - git clone https://github.com/libocca/occa.git
```

Comment:
Let's add `--depth=1` to all the clones. Reduces size from 17M to 600K in this case.

Reply:
Erm, won't work for a specific hash. I guess this https://stackoverflow.com/questions/31278902/how-to-shallow-clone-a-specific-commit-with-depth-1

Reply:
I dropped the `--depth=1` argument on OCCA last year when we were having trouble with it's stability effecting our tests. Do we want to drop the testing against a 'known good' commit of OCCA now that we haven't had that issue in a while?

Reply:
I dropped it for OCCA, but am temporarily pulling more for Nek. See below